src/translator/service.h

```diff
@@ -14,6 +15,7 @@
 #include <thread>
 #endif
 
+#include <optional>
```

Comment:
Do you need `optional` header in this file? Probably, it will go to the file where you are doing the comparison?

Reply:
Commit 6f68978